### PR TITLE
MAP-1607 allow multiple specialist cells to be set for convert to cell

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/LocationResidentialResource.kt [hunk 1]
```diff
@@ -385,7 +385,7 @@ class LocationResidentialResource(
     @field:Max(value = 99, message = "Working capacity cannot be greater than 99")
     @field:PositiveOrZero(message = "Working capacity cannot be less than 0")
     val workingCapacity: Int = 0,
-    @Schema(description = "Used For list", example = "STANDARD_ACCOMMODATION, PERSONALITY_DISORDER", required = false)
+    @Schema(description = "Used For list", example = "[ STANDARD_ACCOMMODATION, PERSONALITY_DISORDER ]", required = false)
```
Comment (jimbali) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/LocationResidentialResource.kt:388]: Should this be `"[ \"STANDARD_ACCOMMODATION\", \"PERSONALITY_DISORDER\" ]"`?

Comment (Mjwillis) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/LocationResidentialResource.kt:388]: Yes!

Comment (Mjwillis) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/LocationResidentialResource.kt]: @jimbali re-submitted

**Review by jimbali — APPROVED** [review]
lgtm, thanks!

**Pull request merged**